tests/behat/features/api/get_scratch_remix_programs.feature

```diff
@@ -0,0 +1,125 @@
+@api
+Feature: Get a remixed program from Scratch
+
+  Background:
+    Given there are users:
+      | name     | password | token      | id |
+      | Catrobat | 12345    | cccccccccc |  1 |
+
+    And there are programs:
```

Comment:
please keep the "lines  | " aligned. ( line 10-19)

Reply:
Done